### ISSUE TITLE
Elgg 3.0 - elgg_get_plugins return deleted plugins

### ISSUE THREAD

**Issue by rohit1290** [Elgg/Elgg#12679]
elgg_get_plugins('all') is returning deleted plugins in the list. It should ideally be returning activated and deactivated plugins only.

What is the issue here:
```
$plugins = elgg_get_plugins('all');
foreach($plugins as $plugin){
  echo $plugin->getManifest()->getVersion();
}
```
getManifest will cause an error - plugin path not found/plugin path does not exists

**Comment by hypeJunction** [comment]
You probably need to flush the cache if you are manually deleting the plugin folder

**Comment by rohit1290** [comment]
Flushing the cache, running the upgrade.. I did both.. After that i opened this ticket.

**Comment by rohit1290** [comment]
I am no longer able to replicate the issue. I believe this was fixed somewhere between 3.0 to 3.1.3. Closing the issue.
